labs/lab-1-html-and-css/desktop-stylesheet.css

```diff
@@ -0,0 +1,131 @@
+/*
+Author: Vikram Badarinath
+Created: 01/23/2018
```

Comment:
Cute, but unnecessary, and in a real workplace, many people would edit this file, and you would always get the blame for their mistakes.